### DISCUSSION TITLE
CookieCloud同步服务出现问题

### DISCUSSION THREAD

**Discussion by StarsWhere** [jxxghp/MoviePilot#1488] [问题咨询]
如题,新版本貌似CookieCloud同步服务出现问题,一直无法同步cookie,哪怕反复检查服务器是否出问题,进行多次验证,再查看更新后,确定是MoviePilot自身的问题.

**Comment by StarsWhere** [comment]
已解决

**Reply by yourwater** [reply]
求问怎么解决的，同样的问题
